### PR TITLE
First PR

### PR DESCRIPTION
**Pull request by jbschiratti**
* Added main file for feature extraction (`feature_extraction.py`)
* Added file for bivariate features (`bivariate.py`) + max cross correlation function
* Tests.

### CODE REVIEW

File: mne_features/bivariate.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Author: Jean-Baptiste Schiratti <jean.baptiste.schiratti@gmail.com>
+#         Alexandre Gramfort <alexandre.gramfort@inria.fr>
+# License: BSD 3 clause
+
+
+import numba as nb
+import numpy as np
+from math import sqrt
+from functools import partial
```
Comment (agramfort) [inline mne_features/bivariate.py]: wrong import order. first most core to periphery packages

File: mne_features/bivariate.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    function.
+
+    Parameters
+    ----------
+    sfreq : float
+        Sampling rate of the data.
+
+    Returns
+    -------
+    bivariate_funcs : dict
```
Comment (agramfort) [inline mne_features/bivariate.py]: dict fo what?

File: mne_features/feature_extraction.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import warnings
```
Comment (agramfort) [inline mne_features/feature_extraction.py]: missing authors and license on top of file

File: mne_features/feature_extraction.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import numpy as np
+from mne_features.univariate import get_univariate_funcs
+from mne_features.bivariate import get_bivariate_funcs
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.pipeline import FeatureUnion
```
Comment (agramfort) [inline mne_features/feature_extraction.py]: wrong import order

File: setup.py [hunk 5]
```diff
@@ -57,5 +57,8 @@ def package_tree(pkgroot):
               'Operating System :: MacOS',
           ],
           platforms='any',
-          packages=package_tree('mne_features'),
+          packages=package_tree('mne_features'), install_requires=['numpy',
+                                                                   'scipy',
+                                                                   'numba',
+                                                                   'sklearn']
```
Comment (agramfort) [inline setup.py]: sklearn -> scikit-learn

**Comment by agramfort** [comment]
ok keep going. great

File: mne_features/bivariate.py (hunk 1 above)
Comment (jbschiratti) [reply mne_features/bivariate.py]: Fixed.

File: mne_features/feature_extraction.py (hunk 3 above)
Comment (jbschiratti) [reply mne_features/feature_extraction.py]: Fixed.

File: mne_features/feature_extraction.py (hunk 4 above)
Comment (jbschiratti) [reply mne_features/feature_extraction.py]: Fixed.

File: setup.py (hunk 5 above)
Comment (jbschiratti) [reply setup.py]: Fixed.

File: mne_features/feature_extraction.py [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+# License: BSD 3 clause
+
+
+import warnings
+import numpy as np
+from sklearn.pipeline import FeatureUnion
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.externals import joblib
+from mne_features.bivariate import get_bivariate_funcs
+from mne_features.univariate import get_univariate_funcs
```
Comment (agramfort) [inline mne_features/feature_extraction.py]: from .univariate

use relative imports except for test files.

**Comment by agramfort** [comment]
I think coverage is not complete:

https://codecov.io/gh/mne-tools/mne-features/src/master/mne_features/bivariate.py

let me activate codecov

**Comment by agramfort** [comment]
try to activate coverage only if USE_NUMBA=OFF

**Comment by agramfort** [comment]
Ok coverage is good now. Great

**Pull request merged**